app/src/main/res/layout/activity_map.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context=".MapActivity">
+
+        <Button
+            android:id="@+id/button_get_localisation"
```

Comment:
Nitpick: I'll make this comment once, but it applies a few times

It's "location" not "localisation". "localization" would be the process of adapting something to some language or culture, not the place where something is. It's a "false friend" to the French "localisation"